doc/ref_model/chapters/chapter04.md

```diff
@@ -90,26 +90,20 @@ This section describes a set of explicit Cloud Infrastructure capabilities and p
 
 | Ref       | Cloud Infrastructure Capability           | Unit   | Definition/Notes                                            |
 |-----------|-------------------------------------------|--------|-------------------------------------------------------------|
-| e.cap.006 | CPU core pinning support                  | Yes/No | Indicates if Cloud Infrastructure supports CPU core pinning |
-| e.cap.007 | NUMA support                              | Yes/No | Indicates if Cloud Infrastructure supports NUMA             |
+| e.cap.006 | CPU pinning                               | Yes/No | Indicates if Cloud Infrastructure supports CPU pinning      |
+| e.cap.007 | NUMA alignment                            | Yes/No | Indicates if Cloud Infrastructure supports NUMA             |
 | e.cap.008 | IPSec Acceleration                        | Yes/No | IPSec Acceleration                                          |
 | e.cap.009 | Crypto Acceleration                       | Yes/No | Crypto Acceleration                                         |
 | e.cap.010 | Transcoding Acceleration                  | Yes/No | Transcoding Acceleration                                    |
 | e.cap.011 | Programmable Acceleration                 | Yes/No | Programmable Acceleration                                   |
-| e.cap.012 | Enhanced Cache Management <sup>1)</sup>   | Yes/No | If supported, L=Lean; E=Equal; X=eXpanded                   |
-| e.cap.013 | SR-IOV over PCI-PT <sup>2)</sup>          | Yes/No | Traditional SR-IOV                                          |
-| e.cap.014 | GPU/NPU <sup>2)</sup>                     | Yes/No | Hardware coprocessor                                        |
-| e.cap.015 | SmartNIC <sup>2)3)</sup>                  | Yes/No | Network Acceleration                                        |
-| e.cap.016 | FPGA/other Acceleration H/W <sup>2)</sup> | Yes/No | Non-specific hardware                                       |
+| e.cap.012 | Enhanced Cache Management                 | Yes/No | If supported, L=Lean; E=Equal; X=eXpanded.  L and X cache policies require CPU pinning to be active. |
+| e.cap.013 | SR-IOV over PCI-PT                        | Yes/No | Traditional SR-IOV. These Capabilities generally require hardware-dependent drivers be injected into workloads, which is prohibited by CNTT principles. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the RM Appendix for the usage policy relevant to any needed hardware Capability of this type.  |
```

Comment:
I think a reference to the [exception](https://github.com/cntt-n/CNTT/blob/master/doc/tech/policies.md#7131-sr-iov-over-pci-pt) is enough instead of re-explining the thing.

Reply:
@CsatariGergely The exception was listed as a note but GSMA requires that be in the text. Any reference would have to be content within the GSMA document